test/unit/editors/communicationmapping.test.ts

```diff
@@ -0,0 +1,90 @@
+import { expect } from '@open-wc/testing';
+import {
+  getConnection,
+  getConnectionIndexOf,
+  getControlBlockConnection,
+  getDataConnection,
+} from '../../../src/triggered/CommunicationMapping.js';
+
+describe('CommunicationMappingPlugin', () => {
+  let doc: Document;
+  beforeEach(async () => {
+    doc = await fetch('public/xml/testfiles/comm-map.scd')
+      .then(response => response.text())
+      .then(str => new DOMParser().parseFromString(str, 'application/xml'));
+  });
+
+  describe('getControlBlockConnection', () => {
+    it('retruns an array of Connection`s for valid inputs of type `ReportControl`', () => {
+      expect(getControlBlockConnection(doc, 'ReportControl')).to.have.length(2);
+      expect(getControlBlockConnection(doc, 'ReportControl')[0]).to.deep.equal({
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7203429)
```suggestion
      expect(getControlBlockConnection(doc, "ReportControl")[0]).to.deep.equal({
```